api-reference/50 Common/utils/utils/getTimeZones(date).md

```diff
@@ -20,7 +20,7 @@ The following code illustrates how to use this method:
 
     const timeZones = DevExpress.utils.getTimeZones(new Date(2020, 6, 21));
     // ===== or when using modules =====
-    import timeZoneUtils from 'devextreme/time_zone_utils';
+    import { timeZoneUtils } from 'devextreme/time_zone_utils';
```

Comment:
The import is named as `getTimeZones`, not `timeZoneUtils`
https://github.com/DevExpress/DevExtreme/blob/21_2/js/time_zone_utils.js#L4